examples/static-site/Caddyfile

```diff
@@ -0,0 +1,5 @@
+0.0.0.0:8080 {
+        root /var/www/public
+        log stdout
+        gzip
```

Comment:
It's pretty generic, and it's just a guideline, every project can adapt to its need later on